### ISSUE TITLE
Instability in training OPANAS from Scratch. 

### ISSUE THREAD

**Issue by cydiachen** [VDIGPKU/OPANAS#8]
Dear Author: 
I am now trying to reproduce your results from training from scratch on COCO Dataset. But it occurs to me that the Model cannot covergence with APEXOptimizer hook. Could you provide me with some advices?

**Comment by tingtingliangvs** [comment]
Hi, thanks for noticing.
The convergence of this model is not very stable due to the deep Neck architecture. In my previous work, I tried lowering the initial learning rate and adding batch normalization in Neck to solve the convergence problem. I have no plans for training from scratch in the near future, but issues are welcomed if you run into other problems.
